ehitchhiking-ui/src/app/shared/enums/TripStatus.ts

```diff
@@ -0,0 +1,5 @@
+export enum TripStatus {
+  active =  0,
```

Comment:
enums should starts from capitalized letter

Reply:
fixed